internal/perms/perms.go

```diff
@@ -17,35 +17,21 @@ package perms
 
 import (
 	"fmt"
-	"os/user"
-	"strconv"
+	"os"
 )
 
 // MyUserAndGroup returns the UID and GID of this process.
 func MyUserAndGroup() (uid uint32, gid uint32, err error) {
-	// Ask for the current user.
-	user, err := user.Current()
-	if err != nil {
-		err = fmt.Errorf("Fetching current user: %w", err)
-		return
-	}
-
-	// Parse UID.
-	uid64, err := strconv.ParseUint(user.Uid, 10, 32)
-	if err != nil {
-		err = fmt.Errorf("Parsing UID (%s): %w", user.Uid, err)
-		return
-	}
+	signed_uid := os.Getuid()
+	signed_gid := os.Getgid()
 
-	// Parse GID.
-	gid64, err := strconv.ParseUint(user.Gid, 10, 32)
-	if err != nil {
-		err = fmt.Errorf("Parsing GID (%s): %w", user.Gid, err)
+	if signed_gid == -1 || signed_uid == -1 {
```

Comment:
Is it possible to add unit tests?

Reply:
One approach is to create a separate executable like build_gcsfuse (but one which only exercises this function) and the invoke that through `exec.Command` as it can take a user:group credential. It certainly won't be a unit test. @vadlakondaswetha  What do you think?

Reply:
you need to add a basic test to test the methods are not throwing exception and not returning -1

Reply:
also if you call setuid/setgid from unit tests what happens?

Reply:
> also if you call setuid/setgid from unit tests what happens?

The calls fail. I've added them in the newly added unit tests for now. PTAL.